From: Jeffrey Clark <h0tw1r3@gmail.com>
Date: Wed, 23 Apr 2014 22:37:43 +0200
Subject: analyze: fix plot with bad y size

systemd-analyze plot > test.svg produces output with all y and height
element attributes equal to zero. This of course causes the resulting
svg to appear blank (zero height). Bug does not affect x86. Looks like
a compiler optimization may be the culprit.

https://github.com/archlinuxarm/PKGBUILDs/issues/815
(cherry picked from commit a213b7e977221ca96bbc1b19a5a879c912ba2488)
(cherry picked from commit 6e150a59cacafbb2c339e0854f449d63ef919a4f)
---
 src/analyze/systemd-analyze.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/analyze/systemd-analyze.c b/src/analyze/systemd-analyze.c
index 317baf9..25f8ad6 100644
--- a/src/analyze/systemd-analyze.c
+++ b/src/analyze/systemd-analyze.c
@@ -41,7 +41,7 @@
 #include "pager.h"
 
 #define SCALE_X (0.1 / 1000.0)   /* pixels per us */
-#define SCALE_Y 20.0
+#define SCALE_Y (20.0)
 
 #define compare(a, b) (((a) > (b))? 1 : (((b) > (a))? -1 : 0))
 
